code-challenges.test.js

```diff
@@ -15,37 +15,126 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+describe("shuffleArray", () => {
+  const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+  const colors2 = [
+    "chartreuse",
+    "indigo",
+    "periwinkle",
+    "ochre",
+    "aquamarine",
+    "saffron",
+  ]
+  it("returns a new array that has removed the first item from the original array and shuffled the rest of the items.", () => {
+    expect(shuffleArray(colors1)).toEqual(expect.arrayContaining(colors1))
+    expect(shuffleArray(colors2)).toEqual(expect.arrayContaining(colors2))
+  })
+})
+
```

Comment:
Your test will certainly work, but it is also not testing for the removal of the first value.